### PR TITLE
Added Features of OpenEBS

### PR DESCRIPTION
**Pull request by hammadmashkoor**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: documentation/source/getting_started/features.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.. _Getting-Started:
+Features
+====================
+OpenEBS offers persistent block storage with the following features:
+
+- Containerized block storage using Docker containers. We call them VSMs or Virtual Storage Machines, a concept similar to k8s PODs.
```
Comment (mahebbar) [inline documentation/source/getting_started/features.rst]: VSMs = volumes

Comment (mahebbar) [reply documentation/source/getting_started/features.rst]: Hi Hammad, can you please close this PR so that I can pick it up?

**Comment by hammadmashkoor** [comment]
I have added that Text as suggested....i was away from my Lappy..

**Comment by mahebbar** [comment]
Thank you, Hammad!

**Comment by nitisuryawanshi** [comment]
@hammadmashkoor thanks for contributing to the first edition of OpenEBS Hackfest. We would like to send you some swag as a thank you note for your contribution. Please fill out the following form so that we can send it your way. Many thanks and keep contributing! https://cloudbyte.agilecrm.com/forms/5126296610799616

**Pull request merged**